### DISCUSSION TITLE
Welcome to devops-dictionary Discussions!

### DISCUSSION THREAD

**Discussion by jackwesleyroper** [jackwesleyroper/devops-dictionary#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by urdvhub23** [comment]
**

Hi! I am self-learning devops for about a year now.

Still struggling with the basics. Can someone help how to progress continuously with a fast pace.

How to crack the devops interview as beginner?

**  
here is my article: https://dev.to/urdevops24/learning-devops-projects-from-beginner-to-advance-59ld

**Reply by jackwesleyroper** [reply]
Hi @urdvhub23 thanks for the suggestions, I'll try and dive into those subjects more in my future blog posts 🚀
